src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/data/ExternalMovementRepository.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.digitalprisonreportingmi.data
+
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.model.ExternalMovement
+
+@Service
+class ExternalMovementRepository {
```

Comment:
It would be good to call this a **Fake**ExternalMovementRepository, just to make it super-clear to people who stumble into the repo.

Reply:
This has now been renamed